starport/services/chain/chain_test.go

```diff
@@ -1,48 +0,0 @@
-package chain
-
-import (
-	"os"
-	"path/filepath"
-	"testing"
-
-	"github.com/docker/docker/pkg/archive"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func TestSourceVersion(t *testing.T) {
-	t.Run("tagged latest commit", func(t *testing.T) {
-		c, err := New(tempSource(t, "testdata/version/mars.v0.2.tar.gz"))
-		require.NoError(t, err)
-
-		assert.Equal(t, "v0.2", c.sourceVersion.tag)
-		assert.Equal(t, "503123b1ac552437c7db3d17f816fd4121ff400d", c.sourceVersion.hash)
-	})
-
-	t.Run("tagged older commit", func(t *testing.T) {
-		c, err := New(tempSource(t, "testdata/version/mars.v0.2-3-gaae48b7.tar.gz"))
-		require.NoError(t, err)
-
-		assert.Equal(t, "v0.2-3-gaae48b7", c.sourceVersion.tag)
-		assert.Equal(t, "aae48b7ffa4991bbe229f0969db8fe8623bf1fd4", c.sourceVersion.hash)
-	})
-}
-
-func tempSource(t *testing.T, tarPath string) (path string) {
-	f, err := os.Open(tarPath)
-	require.NoError(t, err)
-
-	defer f.Close()
-
-	dir, err := os.MkdirTemp("", "")
-	require.NoError(t, err)
-
-	t.Cleanup(func() { os.RemoveAll(dir) })
-
-	require.NoError(t, archive.Untar(f, dir, nil))
```

Comment:
Setting this to true in the untar options should solve it: https://github.com/moby/moby/blob/c55a4ac/pkg/archive/archive.go#L40
https://pkg.go.dev/github.com/moby/moby/pkg/archive#Untar

Reply:
Got it! I've change and commit again.